### ISSUE TITLE
`pastehub-clientSync' exit, when Oeration timed out occur.

### ISSUE THREAD

**Issue by kiyoka** [kiyoka/pastehub#1]
waiting...
Info: posted data from MacOS X.
/usr/local/stow/ruby-1.9.3-p194/lib/ruby/1.9.1/net/http.rb:762:in `initialize': Operation timed out - connect(2) (Errno::ETIMEDOUT)
    from /usr/local/stow/ruby-1.9.3-p194/lib/ruby/1.9.1/net/http.rb:762:in`open'
    from /usr/local/stow/ruby-1.9.3-p194/lib/ruby/1.9.1/net/http.rb:762:in `block in connect'
    from /usr/local/stow/ruby-1.9.3-p194/lib/ruby/1.9.1/timeout.rb:54:in`timeout'
    from /usr/local/stow/ruby-1.9.3-p194/lib/ruby/1.9.1/timeout.rb:99:in `timeout'
    from /usr/local/stow/ruby-1.9.3-p194/lib/ruby/1.9.1/net/http.rb:762:in`connect'
    from /usr/local/stow/ruby-1.9.3-p194/lib/ruby/1.9.1/net/http.rb:755:in `do_start'
    from /usr/local/stow/ruby-1.9.3-p194/lib/ruby/1.9.1/net/http.rb:744:in`start'
    from /usr/local/stow/ruby-1.9.3-p194/lib/ruby/1.9.1/net/http.rb:557:in `start'
    from /usr/local/stow/ruby-1.9.3-p194/lib/ruby/gems/1.9.1/gems/pastehub-0.1.0/lib/pastehub/client.rb:87:in`wait_notify'
    from ./pastehub-clientSync:65:in `syncMain'
    from ./pastehub-clientSync:117:in`block in main'
bash-4.2$

**Comment by kiyoka** [comment]
Same error occurs today.
I shutdown the server machine that serve maseterdb.rb and notifier.rb durling pastehub-clientSync execution.
Then, this error occurs.

```
waiting...
/usr/local/stow/ruby-1.9.3-p194/lib/ruby/1.9.1/net/http.rb:762:in `initialize': Connection timed out - connect(2) (Errno::ETIMEDOUT)
    from /usr/local/stow/ruby-1.9.3-p194/lib/ruby/1.9.1/net/http.rb:762:in `open'
    from /usr/local/stow/ruby-1.9.3-p194/lib/ruby/1.9.1/net/http.rb:762:in `block in connect'
    from /usr/local/stow/ruby-1.9.3-p194/lib/ruby/1.9.1/timeout.rb:54:in `timeout'
    from /usr/local/stow/ruby-1.9.3-p194/lib/ruby/1.9.1/timeout.rb:99:in `timeout'
    from /usr/local/stow/ruby-1.9.3-p194/lib/ruby/1.9.1/net/http.rb:762:in `connect'
    from /usr/local/stow/ruby-1.9.3-p194/lib/ruby/1.9.1/net/http.rb:755:in `do_start'
    from /usr/local/stow/ruby-1.9.3-p194/lib/ruby/1.9.1/net/http.rb:744:in `start'
    from /usr/local/stow/ruby-1.9.3-p194/lib/ruby/1.9.1/net/http.rb:557:in `start'
    from /home/kiyoka/work/github/pastehub/lib/pastehub/client.rb:91:in `wait_notify'
    from bin/pastehub-clientSync:90:in `syncMain'
    from bin/pastehub-clientSync:153:in `block in main'
```

rake aborted!

**Comment by kiyoka** [comment]
This issue was already resolved on 0.1.3
